composer.json

```diff
@@ -56,15 +55,20 @@
         "laminas/laminas-mvc": "~2.7.0",
         "laminas/laminas-serializer": "^2.7.2",
         "laminas/laminas-server": "^2.6.1",
-        "laminas/laminas-servicemanager": "^2.7.8",
+        "laminas/laminas-servicemanager": "^3.5.1",
+        "laminas/laminas-servicemanager-di": "^1.2",
```

Comment:
I see that laminas-servicemanager-di archived and is not required https://github.com/laminas/laminas-servicemanager-di

> You can now use laminas/laminas-di directly with laminas/laminas-servicemanager.